### ISSUE TITLE
One-Click Reblog Not White

### ISSUE THREAD

**Issue by parkr** [jcutler/Missing-E#21]
Screenshot: http://cl.ly/0e1J2G1t3J3p403h2U30

Running latest version (1.4.0)

**Comment by jcutler** [comment]
Browser/OS? Can you try adding this to betterReblogs/quickReblog.css and see if it fixes the issue?

```
#MissingE_quick_reblog .user_menu_list a, #MissingE_quick_reblog .user_menu_list .user_menu_list_item {
    color:white !important;
}
```

**Comment by parkr** [comment]
Sorry, Chrome/Mac. That snippet is a no-go.

**Comment by jcutler** [comment]
I'll have to try it on my Mac this evening.

In the meantime, are you familiar with the DOM inspector in Chrome? I would absolutely love if you could figure out why the colour is wrong (even though the CSS rules should be fine).

Somewhere at the bottom of the DOM is a div with `id="MissingE_quick_reblog"`. Just add to its style `display:block` and you can start looking at it.

Every single node level below that div has `color` set to `white !important`, and that **should** have been enough!

**Comment by parkr** [comment]
From what I can see, the DOM inspector is preferring the color of `a` in `global.css`, yet showing that white was selected (the rest are crossed-out). There tends to be a sensitivity with `a` elements in Chrome. I'll fiddle with it for a minute.

**Comment by jcutler** [comment]
Thanks. Let me know if you figure out anything that works.

**Comment by jcutler** [comment]
This should be fixed now that I'm using #FEFEFE for the font colour instead. Please confirm?

**Comment by parkr** [comment]
Nah, man. :/

![Still that same problem](http://f.cl.ly/items/3L2f32183f452P1S3E0Z/missinge.png)

**Comment by jcutler** [comment]
Can you look into the CSS in the Inspector to see if it is even attempting to use FEFEFE?

**Comment by parkr** [comment]
From what I could see, `!important` was used too much. It seems like it's all working now.

When I went into the DOM Inspector and messed with activating and deactivating the various declarations, I found that deactivating the `color: white !important` helped.
![Deactivated white !important](http://f.cl.ly/items/2Z0k1Y3q2E160Y3l1M1t/missinge-dom.png)

It works! I submitted a pull request for you.
![Finally working!](http://f.cl.ly/items/390117210p2v0D1i2N1c/missinge-working.png)
